test/verify/check-terminal

```diff
@@ -98,6 +98,69 @@ PROMPT_COMMAND='printf "\\033]0;%s@%s:%s\\007" "${USER}" "${HOSTNAME%%.*}" "${PW
         wait_line(n + 1, blank_state)
         self.assertNotIn('admin', b.text(line_sel(n + 1)))
 
+        if m.image == "rhel-7-6-distropkg":
+            return
+
+        def select_line(sel, width):
+            # Select line by dragging mouse
+            # 10px padding on all sides
+            # Height on a line is around 14px, so start approx. in the middle of line
+            b.mouse(sel, "mousedown", 10, 17)
+            b.mouse(sel, "mousemove", 10 + width, 17)
+            b.mouse(sel, "mouseup", 10 + width, 17)
+
+
+        b.grant_permissions("http://%s:%s" % (self.machine.web_address, self.machine.web_port),
```

Comment:
It feels like the origin should be computed internally by `grant_permissions()` -- I don't see us ever using any other origin.